code/controllers/subsystem/air.dm

```diff
@@ -19,6 +19,10 @@ SUBSYSTEM_DEF(air)
 	var/list/generated_atmospheres
 	/// cached lists of unpacked gas-strings
 	var/list/cached_strings
+	/// scrubber defaults
+	var/list/scrubber_defaults = list(
+		SCRUBBER_DEFAULT_STATION = list(/datum/gas/carbon_dioxide, /datum/gas/phoron),
```

Comment:
nope; initial() doesn't work with lists. this uses SSair lists now because we have to resolve the gas ids/whatnot and we don't want to do that on every scrubber.

iirc i did leave in the capability to varedit scrubbers on maps.